moto/mediastoredata/models.py

```diff
@@ -0,0 +1,113 @@
+from __future__ import unicode_literals
+
+import datetime
+from collections import OrderedDict
+
+from boto3 import Session
+
+from moto.core import BaseBackend, BaseModel
+from .exceptions import ClientError
+
+
+class CreatedObject(BaseModel):
+    def __init__(self, *args, **kwargs):
+        self.path = kwargs.get("Path")
+        self.content_sha256 = kwargs.get("ContentSHA256")
+        self.etag = kwargs.get("ETag")
+        self.storage_class = kwargs.get("StorageClass")
+
+    def to_dict(self, exclude=None):
+        data = {
+            "ETag": self.etag,
+            "Name": self.path,
+            "Type": "FILE",
+            "ContentLength": 123,
+            "StorageClass": self.storage_class,
+            "Path": self.path,
+            "ContentSHA256": self.content_sha256,
+        }
+        if exclude:
+            for key in exclude:
+                del data[key]
+        return data
+
+
+class Object(BaseModel):
```

Comment:
This object doesn't seem to be used. (As far as naming goes - I'd prefer `Object` over `CreatedObject` though)